src/Surfnet/StepupRa/RaBundle/Controller/VettingController.php

```diff
@@ -79,15 +79,6 @@ public function startProcedureAction(Request $request)
             return ['form' => $form->createView()];
         }
 
-        if (!$this->isGranted('ROLE_SRAA') && $secondFactor->institution !== $this->getIdentity()->institution) {
```

Comment:
As discussed IRL should we drop these checks? Or enhance them in order to let a user know if access is not granted?

And I was wondering what is happening with the SRAA at the moment?

Reply:
Good point, now that we h ave the FGA context in RA it is once again possible to perform this check.